components/image-builder-api/imgbuilder.proto

```diff
@@ -128,4 +128,6 @@ message BuildInfo {
     BuildStatus status = 2;
     int64 started_at = 3;
     string build_id = 5;
+    string log_url = 6;
+    map<string, string> log_url_extra_headers = 7;
```

Comment:
A matter of taste, but I'd wrap this in a message of its own, e.g.
```protobuf
    LogInfo logs = 6;
}

message LogInfo {
    string url = 1;
    map<string, string> header = 7;
}
```

Reply:
I'd prefer that myself - also to mirror the code that's already there. :+1: